test/classes/initializers/assign-param-segfault.bad

```diff
@@ -0,0 +1,8 @@
+assign-param-segfault.chpl:6: In initializer 'Bar':
+assign-param-segfault.chpl:11: internal error: EXP0492 chpl Version 1.12.0.223383c
+
+Internal errors indicate a bug in the Chapel compiler ("It's us, not you"),
+and we're sorry for the hassle.  We would appreciate your reporting this bug -- 
+please see http://chapel.cray.com/bugs.html for instructions.  In the meantime,
+the filename + line number above may be useful in working around the issue.
+
```

Comment:
Shoot, I assumed that there wouldn't be any others, thanks for catching.  This is why I ask for reviews ;)